e2etests/web/regular_integration_tests/test_driver/treeshaking_e2e.dart

```diff
@@ -0,0 +1,59 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:html' as html;
+import 'package:flutter_test/flutter_test.dart';
+import 'package:regular_integration_tests/treeshaking_main.dart' as app;
+import 'package:flutter/material.dart';
+
+import 'package:e2e/e2e.dart';
+
+void main() {
+  E2EWidgetsFlutterBinding.ensureInitialized() as E2EWidgetsFlutterBinding;
+
+  testWidgets('debug+Fill+Properties for widgets is tree shaken',
+          (WidgetTester tester) async {
+    // About 11 instances are used by DiagnosticsNode itself.
+    await testOccurenceCountBelow(tester, '${debugPrefix}FillProperties', 20);
```

Comment:
I'm afraid the actual number will go down but nobody will remember to update this test. I'm not sure how to prevent this other than making the test check for the exact number instead of "below".

Reply:
Main reason i added this is to prevent a sudden 300+ due to change in diagnostics.dart. 1 up or down is fine.